client/lib/interval/test/interval.tsx

```diff
@@ -1,18 +1,17 @@
 /**
- * @format
  * @jest-environment jsdom
  */
 
 /**
  * External dependencies
  */
-import { mount } from 'enzyme';
 import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
If you're so inclined, perhaps you could give dropping this dependency a go, and using `act` from `react-dom/test-utils` instead?

See https://reactjs.org/docs/test-utils.html#act and e.g. https://github.com/Automattic/wp-calypso/blob/master/client/lib/viewport/test/react.js

Reply:
Thanks for the idea, I was unfamiliar with `act`. I've implemented to changes you suggested if you'd like to take another look 🙇

Reply:
That looks great! Thank you for the extra work! 🙂